.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run Make Image Script
+      run: ./make_img.sh
+    - name: Github Release
+      uses: meeDamian/github-release@v1.0.0
```

Comment:
Add this line to this step:

```
if: startsWith(github.ref, 'refs/tags/')
```

And then above you can change:

```yaml
on:
  push:
    tags:
      - '*'
```

to:

```yaml
on: [push]
```

That would build all pushes, but only add git-tag pushes to :octocat: Releases.

Reply:
cool that works